test/core/integration/fuzz/lockup-base/cancel.t.sol

```diff
@@ -62,48 +62,45 @@ abstract contract Cancel_Integration_Fuzz_Test is Integration_Test {
         lockup.allowToHook(address(recipientGood));
         resetPrank({ msgSender: users.sender });
 
-        // Create the stream.
-        uint256 streamId = createDefaultStreamWithRecipient(address(recipientGood));
-
         // Simulate the passage of time.
         vm.warp({ newTimestamp: defaults.START_TIME() + timeJump });
 
         // Bound the withdraw amount.
-        uint128 streamedAmount = lockup.streamedAmountOf(streamId);
+        uint128 streamedAmount = lockup.streamedAmountOf(recipientGoodStreamId);
         withdrawAmount = boundUint128(withdrawAmount, 0, streamedAmount - 1);
 
         // Make the withdrawal only if the amount is greater than zero.
         if (withdrawAmount > 0) {
-            lockup.withdraw({ streamId: streamId, to: address(recipientGood), amount: withdrawAmount });
+            lockup.withdraw({ streamId: recipientGoodStreamId, to: address(recipientGood), amount: withdrawAmount });
         }
 
         // Expect the assets to be refunded to the Sender.
-        uint128 senderAmount = lockup.refundableAmountOf(streamId);
+        uint128 senderAmount = lockup.refundableAmountOf(recipientGoodStreamId);
         expectCallToTransfer({ to: users.sender, value: senderAmount });
 
         // Expect the relevant events to be emitted.
-        uint128 recipientAmount = lockup.withdrawableAmountOf(streamId);
+        uint128 recipientAmount = lockup.withdrawableAmountOf(recipientGoodStreamId);
         vm.expectEmit({ emitter: address(lockup) });
-        emit ISablierLockup.CancelLockupStream(
-            streamId, users.sender, address(recipientGood), dai, senderAmount, recipientAmount
+        emit ISablierLockupBase.CancelLockupStream(
+            recipientGoodStreamId, users.sender, address(recipientGood), dai, senderAmount, recipientAmount
         );
         vm.expectEmit({ emitter: address(lockup) });
-        emit IERC4906.MetadataUpdate({ _tokenId: streamId });
+        emit IERC4906.MetadataUpdate({ _tokenId: recipientGoodStreamId });
 
         // Cancel the stream.
-        lockup.cancel(streamId);
+        lockup.cancel(recipientGoodStreamId);
```

Comment:
if we use the `recipientGoodStreamId` stream id to test here, why  not expect call to the hook?

Reply:
Yes we should. Also, we can move `lockup.allowToHook` to Integration contract instead of calling in each contract.

Reply:
> Also, we can move lockup.allowToHook to Integration contract instead of calling in each contract.

sorry, not sure what you mean

Reply:
`recipientReverting`, `recipientInvalidSelector`, `recipientReentrant`, `recipientGood` are all expected to be allowed to hook. So we allow them in several tests. Instead of that, we can DRY'ify by allowing them once in `Integration_Test.setUp()` where we define them. Its similar to [this](https://github.com/sablier-labs/v2-core/blob/test/singleton-contract/test/Base.t.sol#L106).